Example/package.json

```diff
@@ -7,8 +7,8 @@
     "adb-reverse": "adb reverse tcp:8081 tcp:8081"
   },
   "dependencies": {
-    "react": "15.4.1",
-    "react-native": "^0.40.0",
+    "react": "15.3.1",
+    "react-native": "^0.33.1",
```

Comment:
This should not be changed. We made a decision a while back to require RN 0.40 or greater.